cmd/talosctl/cmd/talos/etcd.go

```diff
@@ -99,22 +100,28 @@ var etcdMemberListCmd = &cobra.Command{
 					continue
 				}
 
-				if i == 0 {
+				for _, member := range message.Members {
+					if i == 0 {
+						if node != "" {
+							fmt.Fprintln(w, "NODE\tID\tHOSTNAME\tPEER_URLS\tCLIENT_URLS")
+							pattern = "%s\t" + pattern
+						} else {
+							fmt.Fprintln(w, "ID\tHOSTNAME\tPEER_URLS\tCLIENT_URLS")
```

Comment:
`PEER_URLS` might be better as `PEER URLS` ? ans same with `CLIENT_URLS`?

Reply:
yep, fixed